tests/functional/cnv/test_vms.py

```diff
@@ -0,0 +1,128 @@
+import logging
+import pytest
+
+from ocs_ci.framework.testlib import E2ETest
+from ocs_ci.framework.pytest_customization.marks import workloads, magenta_squad
+from ocs_ci.helpers.cnv_helpers import cal_md5sum_vm
+from ocs_ci.helpers.performance_lib import run_oc_command
+from ocs_ci.helpers.keyrotation_helper import PVKeyrotation
+from ocs_ci.utility.utils import run_cmd
+from ocs_ci.ocs import constants
+
+logger = logging.getLogger(__name__)
+
+
+@magenta_squad
+class TestCNVVM(E2ETest):
```

Comment:
@avd-sagare Are we sure to have this test case under `tests/functional/cnv/` instead current cnv workloads dir `tests/functional/workloads/cnv/`? Why separate dirs for similar type of tests(ie workload tier tests)?

Reply:
Actually initially after  discussion with prasad I decided to create  separate dir for integration related tcs.
I will move this to workloads directory.

Reply:
changes done.